daemon/create_windows.go

```diff
@@ -1,11 +1,82 @@
 package daemon
 
 import (
+	"fmt"
+
 	"github.com/docker/docker/image"
+	"github.com/docker/docker/pkg/stringid"
 	"github.com/docker/docker/runconfig"
+	"github.com/docker/docker/volume"
 )
 
 // createContainerPlatformSpecificSettings performs platform specific container create functionality
 func createContainerPlatformSpecificSettings(container *Container, config *runconfig.Config, hostConfig *runconfig.HostConfig, img *image.Image) error {
+	for spec := range config.Volumes {
+		mp, err := volume.ParseMountSpec(spec, hostConfig.VolumeDriver)
```

Comment:
I wonder if there is a way to extract the common code from here and `create_unix.go` leaving out only the part that's not supported on windows. This looks like duplicated code.